easyrest/views/user_controller.py

```diff
@@ -110,3 +111,75 @@ def get_users_list(request):
                   request.dbsession.query(User).filter_by(role_id=derivable_role_id).order_by(User.name).all()]
 
     return wrap(users_list, success=True, message='Users with role \'{}\''.format(role.name))
+
+
+@view_config(route_name='user_create', renderer='json', request_method='POST')
+def create_user(request):
+    """This function is intended to create a user with a specific role.
+
+       This function processes the route /user/{role_id:\d+}
+       and tries to create a user depending on the identifier
+       that is extracted from the parameter {role_id}.
+
+    :param request: POST request
+    :return: If incorrect json:
+                {
+                  "message": null,
+                  "data": [],
+                  "success": false,
+                  "error": "Incorrect json format"
+                }
+             If validation errors:
+                {
+                  "message": null,
+                  "data": [],
+                  "success": false,
+                  "error": "validation_errors"
+                }
+             If user created successfully:
+                {
+                  "message": "User successfully added",
+                  "data": [],
+                  "success": true,
+                  "error": null
+                }
+    :raise HTTPNotFound: If user role id not found.
+    :raise HTTPForbidden: If if an authorization is needed to perform an action.
+    """
+    log = logging.getLogger(__name__)
+
+    try:
+        form_data = request.json_body
+    except ValueError as ve:
+        log.error(ve.message)
+        return wrap([], success=False, error='Incorrect json format')
```

Comment:
Maybe HTTPBadrequest pyramid error is better suited here? It will return response with status 400 and message.

Reply:
You're right. I will replace it.